### DISCUSSION TITLE
River 2 Max - Open ports -which FW works?

### DISCUSSION THREAD

**Discussion by nealjane** [vwt12eh8/hassio-ecoflow#72] [General]
Anyone know what wifi firmware version the River 2 max needs to work/open ports? - Im on v2.1.1.54(wifi), & v4.1.1.43.

Inputting ip into HA and it never find anything - just return an 'unknown error occoured' after a long while

**Comment by eliamontanari** [comment]
Hi, i have the same issue, my firmware version is V3.1.1.88 - V2.1.4.31 (WIFI)
